src/services/schemas/utilities.ts

```diff
@@ -113,6 +114,10 @@ export function getSchemaPropertyPlaceholder(property: SchemaProperty): string |
     return placeholder
   }
 
+  if (isArray(placeholder)) {
+    return placeholder.join(', ')
+  }
```

Comment:
What's this for? Surprised a placeholder could ever be an array

Reply:
It's cause we get the placeholder from the default value and stringify it below this; we've not noticed it cause pselect doesn't use the placeholder anywhere but if it did placeholder values would otherwise display as `["gratis"]`